controllers/productController.js

```diff
@@ -0,0 +1,12 @@
+const productService = require('../services/productService');
+
+const productList = async (req, res) => {
+	const { category } = req.params;
+	const sort = req.query.sort ? req.query.sort : 'id-asc';
+
+	const productList = await productService.productList(category, sort);
+
+	return res.status(200).json({ productList });
```

Comment:
service단에 있던 `sort` split 하는 과정이 controller에 있어야 할 것 같습니다.
`id`, `asc` 구분 하는 과정이 -> request에 있는 parameter를 가공하는 과정이기 때문에,
request를 관리하는 controller에 있는 것이 보다 적합할 것 같습니다.
```suggestion
	const { category } = req.params;
	const sort = req.query.sort ? req.query.sort : 'id-asc';

	const productList = await productService.productList(category, sortingVariable, sortingCondition);

	return res.status(200).json({ productList });
```